client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -59,10 +59,10 @@ object FlightComponents {
     val port: String = if (flight.ActPax > 0) flight.ActPax.toString else "n/a"
     val last: String = flight.LastKnownPax.getOrElse("n/a").toString
     val max: String = if (flight.MaxPax > 0) flight.MaxPax.toString else "n/a"
-
+    val PortTotPax: Int = flight.ActPax + flight.TranPax
```

Comment:
Sorry wasn't aware of the lowercase thing, and now changed so maths should be right